README.md

```diff
@@ -1,2 +1,2 @@
 # 100DaysofOpenSource
-#100DaysofOpenSource - join us here and on Twitter!
+#100DaysofOpenSource - join us here and on [Twitter!](https://twitter.com/ykdojo)
```

Comment:
thank you for the PR!

could you change it to the search page for the hashtag?

this should be more about the hashtag than my personal account

Reply:
```suggestion
#100DaysofOpenSource - join us here and on [Twitter!](https://twitter.com/hashtag/100DaysOfOpenSource?src=hashtag_click)
```